request.go

```diff
@@ -209,5 +209,9 @@ func (client *Client) createRequest(method, api string, reqbody interface{}) (*h
 	if bodyReader != nil {
 		req.Header.Add("Content-Type", "application/json")
 	}
+	for k, v := range client.ExtraHeader {
+		req.Header.Add(k, v)
```

Comment:
It looks like they get appended using curl, I would follow this implementation in case for whatever reason the same key can be used multiple times.

```
➜  ~ curl -H "Content-Type:application/json"  -H "Content-Type:test" -vvv localhost
* Rebuilt URL to: localhost/
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 80 (#0)
> GET / HTTP/1.1
> Host: localhost
> User-Agent: curl/7.54.0
> Accept: */*
> Content-Type:application/json
> Content-Type:test
```

Reply:
Ahh good point 😄 Is the order important here? Should/does extra headers always come at the end?

Reply:
the order is not important because it's sent in the request, and the server can process it however it wants.